### PR TITLE
Scripts for serializing ontology files

### LINKED ISSUE #31: Script for serializing ontology files
Prepare a script that would simplify running the [RDF Toolkit](https://github.com/edmcouncil/rdf-toolkit/#run-rdf-toolkit) on ontology files

### PR DESCRIPTION
**Pull request by przemekgradzki**
Closes #31

### CODE REVIEW

**Review by bszady — DISMISSED** [review]
I have tested and the file (in my opinion) was changed.

File: rdf-toolkit.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# How to start:
+  - on windows: drag the ontology file to the "rdf-toolkit.bat" file
+    or start the `cmd.exe` in the current directory
+    run the `rdf-toolkit.bat` script with the ontology file as an argument, e.g.:
+    ```
+    rdf-toolkit.bat ontology.ttl
+    ```
+  - on Unix-like systems: run the `rdf-toolkit.sh` script with the ontology file as an argument, e.g.:
+    ```bash
+    ./rdf-toolkit.sh ontology.ttl
```
Comment (bernardyn) [inline rdf-toolkit.md:10]: please correct the current ontology name in our repository
```suggestion
    ./rdf-toolkit.sh ontohgis.ttl
```

File: rdf-toolkit.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# How to start:
+  - on windows: drag the ontology file to the "rdf-toolkit.bat" file
+    or start the `cmd.exe` in the current directory
+    run the `rdf-toolkit.bat` script with the ontology file as an argument, e.g.:
+    ```
+    rdf-toolkit.bat ontology.ttl
```
Comment (bernardyn) [inline rdf-toolkit.md:6]: not sure about ontology.ttl to the correct our .ttl file to `ontohgis.ttl`

File: rdf-toolkit.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+# === don't touch anything below
+
+test -n "${1}" || { echo "ERR: missing arg - full path to .ttl file" ; read -n 1 -s -r -p "Press any key to continue...." ; exit 1 ; }
+
+curl --version &>/dev/null || { echo "ERR: missing curl" ; read -n 1 -s -r -p "Press any key to continue...." ; exit 1 ; }
+
+java --version &>/dev/null || { echo "ERR: missing java - download e.g. from https://openjdk.org/install/ and install" ; read -n 1 -s -r -p "Press any key to continue...." ; exit 1 ; }
+
+test -n "${JARP}" || export JARP="${TMP:-/tmp}"
```
Comment (bernardyn) [inline rdf-toolkit.sh:16]: As we discussed to align script to macos please change TMP to TMPDIR in all places

**Review by bernardyn — CHANGES_REQUESTED** [review]

File: rdf-toolkit.sh (hunk 3 above)
Comment (przemekgradzki) [reply rdf-toolkit.sh]: Resolved in commit 0bf817b

File: rdf-toolkit.md (hunk 1 above)
Comment (przemekgradzki) [reply rdf-toolkit.md]: Resolved in commit 49fff4a

File: rdf-toolkit.md (hunk 2 above)
Comment (przemekgradzki) [reply rdf-toolkit.md]: Resolved in commit 49fff4a

**Review by bernardyn — APPROVED** [review]

**Comment by bernardyn** [comment]
@bszady those changes are related only to macos/linux so I think I can merge them without your review.

**Pull request merged**